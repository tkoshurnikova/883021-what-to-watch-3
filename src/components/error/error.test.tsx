import * as React from "react";
import * as renderer from "react-test-renderer";
import Error from "./error";


it(`Render Error`, () => {
  const tree = renderer
    .create(<Error/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
