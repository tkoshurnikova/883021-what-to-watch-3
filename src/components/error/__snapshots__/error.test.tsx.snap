// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Error 1`] = `
<div
  className="user-page"
>
  <h1>
    Sorry, something went wrong!
  </h1>
  <p>
    Please, try again later
  </p>
</div>
`;
